modules/carousel/jetpack-carousel.php

```diff
@@ -212,6 +212,24 @@ function enqueue_assets( $output ) {
 			}
 
 			/**
+			 * Handle WP stats for images in full-screen.
+			 * Build string with tracking info.
+			 */
+			if ( ! in_array( 'stats', Jetpack::get_active_modules() ) ) {
```

Comment:
Since Carousel can be used in [Development mode](https://jetpack.com/support/development-mode/), it might be worth adding a check for development mode here, with `is_development_mode()` ([#](https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L1490))